lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -35,6 +36,9 @@
     __chakraLibrary.ArrayIterator.prototype = CreateObject(iteratorPrototype);
     __chakraLibrary.raiseNeedObjectOfType = platform.raiseNeedObjectOfType;
     __chakraLibrary.raiseThis_NullOrUndefined = platform.raiseThis_NullOrUndefined;
+    __chakraLibrary.raiseFunctionArgument_NeedFunction = platform.raiseFunctionArgument_NeedFunction;
+    __chakraLibrary.callInstanceFunc = platform.builtInCallInstanceFunction;
+    __chakraLibrary.functionBind = platform.builtInJavascriptFunctionEntryBind;
```

Comment:
What exactly is the goal here? Why does `__chakraLibrary` need these if you already have access to them through `platform`

Reply:
We don't allow access to any functions that are not on __chakraLibrary from within the built in functions.